registry.go

```diff
@@ -76,6 +81,12 @@ func IsListenerRegistered(typez string) bool {
 	return found
 }
 
+func DescribeListener(typez string) ([]describe.Parameter, error) {
```

Comment:
I think here is missing listenersLock Mutex, and deal with not being found could be trivial depending on what would be expected to be returned to the CLI.